pkgs/development/libraries/totem-pl-parser/default.nix

```diff
@@ -16,8 +16,16 @@ stdenv.mkDerivation rec {
     };
   };
 
-  nativeBuildInputs = [ meson ninja pkg-config gettext gobject-introspection ];
-  buildInputs = [ libxml2 ];
+  strictDeps = true;
+  nativeBuildInputs = [ meson ninja pkg-config gettext glib ];
+  buildInputs = [ libxml2 glib ]
+    ++ lib.optionals (stdenv.buildPlatform == stdenv.hostPlatform) [
+    gobject-introspection
```

Comment:
I am more confused why is this no longer in `nativeBuildInputs`. Should not this be used for `g-ir-compiler`/`g-ir-scanner` tools at build time?

Reply:
Yes indeed, it just required pkgsBuildBuild = [ pkg-config ]; instead of this. Not sure why this also works though...